source/installation_and_configuration/frontend_installation/install.rst

```diff
@@ -252,7 +252,7 @@ The complete list of operating system services provided by OpenNebula:
 | **opennebula-ssh-socks-cleaner**      | Periodic cleaner of SSH persistent connections                         | opennebula                |
 +---------------------------------------+------------------------------------------------------------------------+---------------------------+
 
-.. note:: Since 5.12, the OpenNebula comes with an integrated SSH agent as the ``opennebula-ssh-agent`` service which removes the need to copy oneadmin's SSH private key across your Hosts. For more information, you can look at the :ref:`passwordless login <kvm_ssh>` section of the manual. You can opt to disable this service and configure your environment the old way.
+.. note:: Since 5.12, the OpenNebula comes with an integrated SSH agent as the ``opennebula-ssh-agent`` service which removes the need to copy oneadmin's SSH private key across your Hosts. For more information, you can look at the :ref:`passwordless login <kvm_local>` section of the manual. You can opt to disable this service and configure your environment the old way.
```

Comment:
" For more information, you can look at" - If I have vision issues, I cannot look at or see anything, potentially.
Better: "For more information, refer to"